src/nft/components/bag/BagFooter.tsx

```diff
@@ -71,17 +79,25 @@ const WarningText = styled(ThemedText.BodyPrimary)`
   color: ${({ theme }) => theme.accentWarning};
   display: flex;
   justify-content: center;
-  margin: 12px 0 !important;
+  margin-bottom: 10px !important;
+  text-align: center;
+`
+
+const HelperText = styled(ThemedText.Caption)<{ $color: string }>`
+  color: ${({ $color }) => $color};
+  display: flex;
+  justify-content: center;
   text-align: center;
+  margin-bottom: 10px !important;
 `
 
 const CurrencyInput = styled(Row)`
   gap: 8px;
   cursor: pointer;
 `
 
-const PayButton = styled(Row)<{ disabled?: boolean }>`
-  background: ${({ theme }) => theme.accentAction};
+const PayButton = styled.button<{ $backgroundColor: string }>`
```

Comment:
Missing gap between spinner and text. Content is also no longer centered.
![Screen Shot 2023-02-08 at 15 13 09 ](https://user-images.githubusercontent.com/11512321/217673159-be0a170a-d08b-49ee-8600-0e2630f6bae3.png)